OWML.ModHelper.Menus/ModComboInput.cs

```diff
@@ -26,71 +35,57 @@ public ModComboInput(TwoButtonToggleElement element, IModMenu menu, IModInputMen
             Button.OnClick += Open;
             var noButton = ToggleElement.GetValue<Button>("_buttonFalse");
             noButton.transform.parent.gameObject.SetActive(false);
-            _layoutGroup = Button.LayoutGroup;
-
-            var parent = Button.Button.transform.parent;
```

Comment:
I think you should keep this one, since you're repeating `Button.Button.transform.parent` many times

Reply:
K